services/checks/checks_test.go

```diff
@@ -156,7 +252,10 @@ func TestSTTMetrics(t *testing.T) {
 }
 
 func TestVerifySignatures(t *testing.T) {
+	t.Parallel()
+
 	t.Run("normal", func(t *testing.T) {
+		t.Parallel()
```

Comment:
Please add a blank line there (like above and below)

Reply:
Done